base/applications/mspaint/drawing.cpp

```diff
@@ -289,34 +289,84 @@ Text(HDC hdc, LONG x1, LONG y1, LONG x2, LONG y2, COLORREF fg, COLORREF bg, LPCT
 }
 
 BOOL
-ColorKeyedMaskBlt(HDC hdcDest, int nXDest, int nYDest, int nWidth, int nHeight,
-                  HDC hdcSrc, int nXSrc, int nYSrc, HBITMAP hbmMask, int xMask, int yMask,
-                  DWORD dwRop, COLORREF keyColor)
+ColorKeyedMaskBlt(HDC hdcDest, INT nXDest, INT nYDest, INT nWidth, INT nHeight,
```

Comment:
What's the benefit of using the macro INT over the CPP-built-in int here? That's just a question, no request for change yet.

Reply:
Reverted.